components/Transactions/components/TxSummaryLine.tsx

```diff
@@ -15,7 +15,7 @@ import moment from 'moment';
 type TxSummaryLineProps = {
   month: string;
   tx: Transaction;
-  setTxDetail: React.Dispatch<React.SetStateAction<Transaction | null>>;
+  setTxDetail: React.Dispatch<React.SetStateAction<Transaction>>;
```

Comment:
I probably missed it, but does this mean there's also a change where many instances of `?` can be removed?

Reply:
Well, my first goal is/was to remove `null` as a valid value of everywhere, and in this case the value of this function could be `Transaction` or `null`. And now only accepts `Transaction`.